op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -157,7 +156,9 @@ func applyStateMigrationChanges(config *genesis.DeployConfig, l2Allocs types.Gen
 	}
 
 	if migrationBlockTime == 0 {
-		migrationBlockTime = uint64(time.Now().Unix())
+		// If the migration block time is not set, use the time of the last block incremented by one.
+		// This makes sure the migration is deterministic.
+		migrationBlockTime = header.Time + config.L2BlockTime
```

Comment:
I don't think we want to do this, or at least not just one block time, because the sequencer will try to fill all the time between the migration block time and 'now' with blocks when it is started. So it's better to leave some space for the migration to happen, ideally we can pick the time to coincide with the completion of the migration.

Reply:
So to clarify, we would increment header.time by a number roughly equal (maybe slightly less) than the time it takes to the migration so that the node doesn't try to add too many blocks to fill the gap?

Reply:
Yeah so we don't end up spamming blocks just as the network starts, so I would either use the flag or hardcode this a few days before we do the next migration, after we've done a trial run and have an idea of how long it would take.